rest/data/buildlogger_test.go

```diff
@@ -10,9 +10,43 @@ import (
 	"github.com/evergreen-ci/cedar"
 	"github.com/evergreen-ci/cedar/model"
 	"github.com/evergreen-ci/pail"
+	"github.com/mongodb/amboy/queue"
+	"github.com/pkg/errors"
 	"github.com/stretchr/testify/suite"
 )
 
+const testDBName = "cedar_rest_data_test"
+
+func init() {
```

Comment:
Worrying that this needed to be moved. I'm guessing this test was depending on the setup from the other test?

Reply:
Yes, exactly